src/ess/livedata/dashboard/widgets/configuration_widget.py

```diff
@@ -106,16 +109,91 @@ def _create_source_selector(self) -> pn.widgets.MultiChoice:
             margin=(0, 0, 0, 0),
         )
 
+    def _create_aux_source_selectors(self) -> dict[str, pn.widgets.Select]:
+        """Create auxiliary source selection widgets."""
+        selectors = {}
+        size = len(self._config.aux_source_names)
+        for i, (category, options) in enumerate(self._config.aux_source_names.items()):
+            if not options:
+                raise ValueError(
+                    f"Empty options list for aux source category '{category}'"
+                )
+
+            selector = pn.widgets.Select(
+                name=category,
+                options=options,
+                value=options[0],  # Default to first entry
+                sizing_mode='stretch_width',
+                # No left margin and no right margin for last item: Align with source
+                # selector above.
+                margin=(0, 0 if i == size - 1 else 10, 10, 0),
```

Comment:
Not part of this PR. But scattering style information throughout the codebase can make it difficult to adjust the layout later. Is it possible to extract this into a CSS file?